scripts/setup_load_env.sh

```diff
@@ -22,23 +22,23 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-export APP_SQLALCHEMY_DATABASE_URI="postgresql://zenodo:zenodo@localhost/zenodo"
+export APP_SQLALCHEMY_DATABASE_URI="postgresql://localhost/zenodo"
 export ZENODO_DB_HOST="localhost"
 export ZENODO_DB_NAME="zenodo"
-export ZENODO_DB_USERNAME="zenodo"
+export ZENODO_DB_USERNAME=`whoami`
 export ZENODO_DB_PORT="5432"
-export ZENODO_DATA_DIR=$HOME/data/zenodo
+export ZENODO_DATA_DIR=$HOME/src/zenodo-datamigration
 export ZENODO_DUMPS_DIR=$ZENODO_DATA_DIR/dumps
 export ZENODO_FIXTURES_DIR=$ZENODO_DATA_DIR/fixtures
 export ZENODO_PGDUMPS_DIR=$ZENODO_DATA_DIR/pgdumps
 zenodo_pgload()
 {
     ZENODO_DB_SNAPSHOP_FILENAME=$1
     echo "Loading Zenodo DB snapshot from" $ZENODO_DB_SNAPSHOP_FILENAME
-    zenodo db destroy --yes-i-know; zenodo db init; psql --dbname="${ZENODO_DB_NAME}" --host="${ZENODO_DB_HOST}" --port="${ZENODO_DB_PORT}" --username="${ZENODO_DB_USERNAME}" -f $ZENODO_DB_SNAPSHOP_FILENAME
+    zenodo db destroy --yes-i-know; zenodo db init; gunzip -c $ZENODO_DB_SNAPSHOP_FILENAME | psql --dbname="${ZENODO_DB_NAME}" --host="${ZENODO_DB_HOST}" --port="${ZENODO_DB_PORT}" --username="${ZENODO_DB_USERNAME}"
```

Comment:
I found zipped dumps to be much slower to load than raw sql, perhaps we'd rather prefer to save time than space ?